### DISCUSSION TITLE
Does it exist some Rome with React demos?

### DISCUSSION THREAD

**Discussion by iam-frankqiu** [rome/tools#1234] [General]
Does it exist some Rome with react demos?  How can I find it?

**Comment by ematipico** [comment]
We don't have any demos at the moment. Rome would work out of the box with React because it understands JSX out of the box. What would like to have/see?

**Reply by iam-frankqiu** [reply]
> We don't have any demos at the moment. Rome would work out of the box with React because it understands JSX out of the box. What would like to have/see?

Can it understand the VUE file out of the box?

**Reply by ematipico** [reply]
> > We don't have any demos at the moment. Rome would work out of the box with React because it understands JSX out of the box. What would like to have/see?
> 
> Can it understand the VUE file out of the box?

No it can't at the moment, and honestly I don't know how and when we plan to support it. Vue is also a compiler so it might make sense to integrate it in rome via plugin or something
